superdesk/commands/data_updates.py

```diff
@@ -76,7 +76,7 @@ def get_data_updates_files(strip_file_extension=False):
     # create folder if doens't exist
     for folder in get_dirs():
         if not os.path.exists(folder):
-            os.makedirs(folder)
+            continue
```

Comment:
it has even `exists_ok` param that would ignore that error, but I would rather not create anything in `get_` method, so will update this to create those when writing the file